src/platform_impl/web/event_loop/window_target.rs

```diff
@@ -231,6 +231,62 @@ impl<T> WindowTarget<T> {
             runner.request_redraw(WindowId(id));
         });
 
+        let runner = self.runner.clone();
+        canvas.on_pointer_move(move |device_id, location| {
+            runner.send_event(Event::WindowEvent {
+                window_id: WindowId(id),
+                event: WindowEvent::Touch(Touch {
+                    id: device_id as u64,
+                    device_id: DeviceId(device::Id(device_id)),
+                    phase: TouchPhase::Moved,
+                    force: None, // Todo
```

Comment:
It is currently possible to implement the force on Web for Apple Devices ([doc](https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/SafariJSProgTopics/RespondingtoForceTouchEventsfromJavaScript.html)). I think it can be nice to have a complete implementation.

Reply:
I think this is currently out of scope for this PR, but feel free to add it in another one!